app/views/split_checkout/_payment.html.haml

```diff
@@ -1,5 +1,5 @@
 .medium-6#checkout-payment-methods
-  - if @order.distributor.vouchers.present?
+  - if feature?(:vouchers, spree_current_user) && @order.distributor.vouchers.present?
```

Comment:
For context, we though that wasn't necessary as unless you have access to Vouchers in backoffice (which is behind a feature toggle.),  a user wouldn't be able to add a voucher to an enterprise, so the voucher section wouldn't show on the payment step.